src/odemis/acq/stream/test/stream_sparc2_fplm_test.py

```
@@ -0,0 +1,145 @@
+logging.getLogger().setLevel(logging.DEBUG)
+
+CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
+SPARC2_FPLM_CONFIG = CONFIG_PATH + "sim/sparc2-fplm-sim.odm.yaml"
+
+
+class SPARC2TestCaseFPLM(BaseSPARCTestCase):
+    """
+    This test case is specifically targeting the FPLM systems, with PL acquisition
+    """
+    simulator_config = SPARC2_FPLM_CONFIG
+    capabilities = {"spec", "fplm"}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)